zebra-rpc/src/methods/tests/snapshot/snapshots/get_block_subsidy_excessive_height@testnet_10.snap

```diff
@@ -3,7 +3,10 @@ source: zebra-rpc/src/methods/tests/snapshot/get_block_template_rpcs.rs
 expression: get_block_subsidy
 ---
 {
-  "fundingstreams": [],
+  "lockboxstreams": [],
```

Comment:
After the lockbox streams period ends and empty `lockboxstreams` array will be present. No `fundingstreams` array will be there. We need to make sure `zcashd` implements the same behaviour.

Reply:
It looks like zcashd will only have the fields for block heights that include a funding stream or lockbox stream, see: https://github.com/zcash/zcash/pull/6912/files#diff-ccc24453c13307f815879738d3bf00eec351417537fbf10dde1468180cacd2f1R1008-R1013.

Reply:
Fixed as part of https://github.com/ZcashFoundation/zebra/pull/8742/commits/e85d0f32c0c14a396d1faa408abbc613824eb73e